app/javascript/src/hyacinth_v1/components/digital_objects/children/Children.js

```diff
@@ -1,31 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Card } from 'react-bootstrap';
+import { Card, Button } from 'react-bootstrap';
 import { useQuery } from '@apollo/react-hooks';
+import { Link } from 'react-router-dom';
 
 import DigitalObjectInterface from '../DigitalObjectInterface';
 import DigitalObjectList from '../DigitalObjectList';
 import AssetNew from '../new/AssetNew';
 import TabHeading from '../../shared/tabs/TabHeading';
-import { getChildStructureDigtialObjectQuery } from '../../../graphql/digitalObjects';
+import { getChildStructureDigitalObjectQuery, getMinimalDigitalObjectWithProjectsQuery } from '../../../graphql/digitalObjects';
 import GraphQLErrors from '../../shared/GraphQLErrors';
+import { digitalObjectAbility } from '../../../utils/ability';
 
 const Children = (props) => {
   const { id } = props;
 
+  const {
+    loading: minimalObjectLoading,
+    error: minimalObjectError,
+    data: minimalObjectData,
+  } = useQuery(getMinimalDigitalObjectWithProjectsQuery, {
+    variables: { id },
+  });
+
```

Comment:
```suggestion
```
I complicated things with my recent PR by also modifying this file, but now you can actually avoid the extra query for the minimal digital object and just use the `digitalObject` variable in its place in the canReorder check.